src/corona_hakab_model/generation/circles.py

```diff
@@ -1,11 +1,13 @@
 from generation.connection_types import ConnectionTypes
 from dataclasses import dataclass
+import pandas as pd
 
 class Circle:
-    __slots__ = "kind", "agent_count"
+    __slots__ = "kind", "agent_count", "circle_id"
 
-    def __init__(self):
+    def __init__(self,circle_id=-1):
         self.agent_count = 0
+        self.circle_id = circle_id
```

Comment:
Why is this optional?

Reply:
currently not every circle needs an id, and assigning a unique id to every circle is a pretty daunting task. This might be the way to go in the future, but it doesn't really solve any current problems, since the id's are really only useful for exporting to humans, and might create new ones.